### PR TITLE
UITOOL-589/mvp toast

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-589

Before:

image

image

After:

image

image

### Potential Breaking Changes

Should be none

### Acceptance Criteria

-   [X] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react/src/components/toast/Toast.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import classNames from 'classnames';
-import {ReactNode, FC, ComponentClass, useState, useEffect, createElement} from 'react';
+import {FunctionComponent} from 'enzyme';
```
Comment (aiheon) [inline packages/react/src/components/toast/Toast.tsx:2]: wtf

**Review by toofff — APPROVED** [review]
After the fix of @gdostie, it's ok for me

**Comment by aiheon** [comment]
Out of scope but I found my new spinner pretty so I put it in the searchBar also 🙃 

https://user-images.githubusercontent.com/63734941/167696795-e2da6895-b5d4-4fdf-985c-f40e6cd66262.mov

**Review by gdostie — COMMENTED** [review]
Can the spinner be extracted into a reusable style? That would avoid having 2 times the same big bloc of rules both for the download toast and the search bar. It would also make it easier to put it somewhere else if needed.

**Comment by aiheon** [comment]
> Can the spinner be extracted into a reusable style? That would avoid having 2 times the same big bloc of rules both for the download toast and the search bar. It would also make it easier to put it somewhere else if needed.

Yeah, i'll do that in another pr! 😄
https://coveord.atlassian.net/browse/UITOOL-677

File: packages/react/src/components/toast/Toast.tsx [hunk 2]
```diff
@@ -25,15 +24,15 @@ export interface IToastProps {
...
     dismissible?: boolean;
     /**
      * Whether the toast has an animation upon entry
      */
     animate?: boolean;
     /**
-     * The toast will be smaller in size when true
+     * @deprecated Do not use - will have no effect
      */
     isSmall?: boolean;
```
[the remark this answers is not in the document]
Comment (toofff) [reply packages/react/src/components/toast/Toast.tsx:37]: When browsing a complete feature it's easier to see the warnings in the console than to open all the feature files in your IDE.

After that is only a proposal, not a specific request for this pull request ;)

[the remark this answers is not in the document]
Comment (aiheon) [reply packages/react/src/components/toast/Toast.tsx:37]: You can create an idea Jira in our backlog if you want 😄

**Pull request merged**